momepy/coins.py

```diff
@@ -126,7 +130,7 @@ def _split_lines(self):
                 out_line.append(
                     [
                         part,
-                        _compute_orientation(part),
+                        [],
```

Comment:
since the COINS algorithm operates with position-based (index-based) attributes of lines, the very first attribute being "orientation": this is now obsolete, so this list will always be empty (we're not computing orientation anymore), but keeping it there nevertheless for the moment not to mess up with the index-based accessing of attributes in the rest of the algorithm. (would probably require major refactoring if we wanted to get rid of this issue, for now i guess it's best just not to touch it)